cache-tree.c

```diff
@@ -777,7 +777,8 @@ int write_index_as_tree(struct object_id *oid, struct index_state *index_state,
 
 static void prime_cache_tree_rec(struct repository *r,
 				 struct cache_tree *it,
-				 struct tree *tree)
+				 struct tree *tree,
+				 struct strbuf *tree_path)
```

Comment:
These changes to `prime_cache_tree_rec()` seem premature for this commit. We are not using `tree_path` in any interesting capacity yet.

Reply:
Thanks for catching this! The `prime_cache_tree_rec` changes are now in the mixed reset commit.